lib/trento/clusters/events/legacy/host_added_to_cluster.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Trento.Domain.Events.HostAddedToCluster do
+  @moduledoc """
+  This event is emitted when a host is added to a cluster
+  """
+
+  use Trento.Event
```

Comment:
I guess this was moved to `Trento.Support.Event`. That should be why compilation fails :eyes: